### PR TITLE
Add content of the todo statement to message #478

### LINKED ISSUE #478: Can the todo rule put the content of the todo statement in the warning?
It'd be incredibly helpful to have the text immediately after a todo statement be placed in the warning, so you don't have to go into each warning to find out what the todo is about.

### PR DESCRIPTION
**Pull request by masters3d**
closes #478

### CODE REVIEW

File: Source/SwiftLintFramework/Rules/TodoRule.swift [hunk 1]
```diff
@@ -41,14 +41,49 @@ public struct TodoRule: ConfigurationProviderRule {
         ]
     )
 
+    // swiftlint:disable conditional_binding_cascade
```
Comment (jpsim) [inline Source/SwiftLintFramework/Rules/TodoRule.swift]: this rule was removed, so I'd rather not add this, only to have to remember to remove it at some point down the line.

**Comment by jpsim** [comment]
Other than the comment which I don't think should be added, 👍.

**Comment by jpsim** [comment]
This also needs a rebase.

**Comment by masters3d** [comment]
Please use github's `squash and merge`
I tried squashing to one commit but then I get branch conflicts with `CHANGELOG`

Also Travis is failing because brew's version is not updated.
```
  testSwiftLintLints, failed - Repeated `let` statements in conditional binding cascade should be avoided.
```

**Comment by jpsim** [comment]
The diff shows that something went wrong during merging or rebasing. You'll need to address that first.

I wonder if there's an elegant solution to address Travis having an older version installed...

**Comment by masters3d** [comment]
Yeah, I butchered this. I am going to close it for now

**Pull request closed without merging**